janusgraph-dist/src/assembly/static/conf/gremlin-server/janusgraph-cql-es-server.properties

```diff
@@ -1,161 +0,0 @@
-# to 0).
-#
-# Default:    10000
-# Data Type:  Long
-# Mutability: GLOBAL_OFFLINE
-#
-# Settings with mutability GLOBAL_OFFLINE are centrally managed in JanusGraph's
-# storage backend.  After starting the database for the first time, this
-# file's copy of this setting is ignored.  Use JanusGraph's Management System
-# to read or modify this value after bootstrapping.
-cache.db-cache-time = 180000
-
-# Size of JanusGraph's database level cache.  Values between 0 and 1 are
-# interpreted as a percentage of VM heap, while larger values are
-# interpreted as an absolute size in bytes.
-#
-# Default:    0.3
-# Data Type:  Double
-# Mutability: MASKABLE
-cache.db-cache-size = 0.25
-
-# Connect to an already-running ES instance on localhost
-
-# The indexing backend used to extend and optimize JanusGraph's query
-# functionality. This setting is optional.  JanusGraph can use multiple
-# heterogeneous index backends.  Hence, this option can appear more than
-# once, so long as the user-defined name between "index" and "backend" is
-# unique among appearances.Similar to the storage backend, this should be
-# set to one of JanusGraph's built-in shorthand names for its standard index
-# backends (shorthands: lucene, elasticsearch, es, solr) or to the full
-# package and classname of a custom/third-party IndexProvider
-# implementation.
-#
-# Default:    elasticsearch
-# Data Type:  String
-# Mutability: GLOBAL_OFFLINE
-#
-# Settings with mutability GLOBAL_OFFLINE are centrally managed in JanusGraph's
-# storage backend.  After starting the database for the first time, this
-# file's copy of this setting is ignored.  Use JanusGraph's Management System
-# to read or modify this value after bootstrapping.
-index.search.backend=elasticsearch
-
-# The hostname or comma-separated list of hostnames of index backend
-# servers.  This is only applicable to some index backends, such as
-# elasticsearch and solr.
-#
-# Default:    127.0.0.1
-# Data Type:  class java.lang.String[]
-# Mutability: MASKABLE
-index.search.hostname=127.0.0.1
-
-# The Elasticsearch node.client option is set to this boolean value, and
-# the Elasticsearch node.data option is set to the negation of this value.
-# True creates a thin client which holds no data.  False creates a regular
-# Elasticsearch cluster node that may store data.
-#
-# Default:    true
-# Data Type:  Boolean
-# Mutability: GLOBAL_OFFLINE
-#
-# Settings with mutability GLOBAL_OFFLINE are centrally managed in JanusGraph's
-# storage backend.  After starting the database for the first time, this
-# file's copy of this setting is ignored.  Use JanusGraph's Management System
-# to read or modify this value after bootstrapping.
-index.search.elasticsearch.client-only=true
-
-# Or start ES inside the JanusGraph JVM
-#index.search.backend=elasticsearch
-#index.search.directory=db/es
-#index.search.elasticsearch.client-only=false
-#index.search.elasticsearch.local-mode=true
```

Comment:
These properties are invalid and `client-only` mode isn't supported for ElasticSearch for quite some time. These properties will be just ignored. After removing them, this file will be identical to `conf/janusgraph-cql-es.properties`.
Thus, removing this file.